src/main/java/org/junitpioneer/jupiter/DefaultTimeZoneExtension.java

```diff
@@ -43,6 +44,10 @@ public void beforeEach(ExtensionContext context) {
 	private void setDefaultTimeZone(Store store, DefaultTimeZone annotation) {
 		storeDefaultTimeZone(store);
 		TimeZone configuredTimeZone = TimeZone.getTimeZone(annotation.value());
+		if (configuredTimeZone.equals(TimeZone.getTimeZone("GMT")) && !annotation.value().equals("GMT")) {
```

Comment:
I ask you to to add a code comment here (and in the documentation (`.adoc` file)) , because I struggled some time to understand this line. It got clear, when I read the Javadoc of `TimeZone.getTimeZone(...)` which points out

> returns the specified TimeZone, or the GMT zone if the given ID cannot be understood.